packages/deployment-service/src/utils/decode.token.ts

```diff
@@ -0,0 +1,10 @@
+import jwt from 'jsonwebtoken'
+
+export const decodeToken = (
```

Comment:
We already have one of these - it also validates the token! See above

Reply:
deleted